source/common/runtime/runtime_features.cc

```diff
@@ -100,6 +100,7 @@ RUNTIME_GUARD(envoy_reloadable_features_validate_connect);
 RUNTIME_GUARD(envoy_reloadable_features_validate_grpc_header_before_log_grpc_status);
 RUNTIME_GUARD(envoy_reloadable_features_validate_upstream_headers);
 RUNTIME_GUARD(envoy_reloadable_features_xdstp_path_avoid_colon_encoding);
+RUNTIME_GUARD(envoy_restart_features_address_factory_lazy_create_address_string);
```

Comment:
Is anything guarded by this new guard? (I'm not seeing it at the moment)

Reply:
In InstanceFactory::createInstancePtr().